lib/utilities/reportHandler.js

```diff
@@ -39,6 +39,7 @@ function getCapabilities() {
         locationTypeSproxyd: true,
         locationTypeNFS: true,
         locationTypeCephRadosGW: true,
+        locationTypeHyperdrive: true,
```

Comment:
We may have over-engineered this but as practice for incomatible format upgrades, we went ahead with full-v2 breakage :) So the Orbit stack will check for the `locationTypeHyperdriveV2` capability in order to show the new version of the hyperdrive settings.

Reply:
thx, will fix